packages/trpc/server/routers/viewer/teams/create.handler.ts

```diff
@@ -51,25 +60,6 @@ export const createHandler = async ({ ctx, input }: CreateOptions) => {
     return duplicatedRequest;
   }
 
-  let parentId: number | null = null;
-  // If the user in session is part of an org. check permissions
-  if (ctx.user.organization?.id) {
-    if (!isOrganisationAdmin(ctx.user.id, ctx.user.organization.id)) {
-      throw new TRPCError({ code: "UNAUTHORIZED" });
-    }
-
-    const nameCollisions = await prisma.user.findFirst({
-      where: {
-        organizationId: ctx.user.organization.id,
-        username: slug,
-      },
-    });
-
-    if (nameCollisions) throw new TRPCError({ code: "BAD_REQUEST", message: "team_slug_exists_as_user" });
```

Comment:
Why is this code removed? Isn't it preventing the team slug to be same as user slug?